lib/isolator/orm_adapters/active_record_adapter.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Isolator
+  module ActiveRecordAdapter
+    module IsolatedTransactions
+      def self.extended(base)
+        class << base
+          alias_method :transaction_without_isolator, :transaction
+          alias_method :transaction, :transaction_with_isolator
```

Comment:
What about other methods? E.g. `begin_transaction`; maybe, something else.

And the most difficult one: `ActiveRecord::Base.connection.execute 'begin;'`

Reply:
I didn't think it is supposed to work for  `ActiveRecord::Base.connection.execute 'begin;' ` to be honest. I'll figure something

Reply:
Yeah, that's gonna be tricky. Let's postpone this case.

AFAIK ActiveRecord itself tracks open transactions, we can rely on this functionality.